.github/workflows/static.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: PHPStan
-        uses: docker://oskarstark/phpstan-ga
+        uses: OskarStark/phpstan-ga0.12.25
```

Comment:
Missing `@` caused GitHub to stop processing this file.

Reply:
Oops. I made a typo
=)

Thank you